--- a/src/Makefile.in
+++ b/src/Makefile.in
@@ -698,7 +698,7 @@
 ifeq ($(HAVE_PDUMPER),yes)
 	$(AM_V_at)$(MAKE_PDUMPER_FINGERPRINT) $@.tmp
 ifeq ($(DO_CODESIGN),yes)
-	codesign -s - -f $@.tmp
+	/usr/bin/codesign -s - -f $@.tmp
 endif
 endif
 	$(AM_V_at)mv $@.tmp $@
@@ -901,7 +901,7 @@
 	@: In the pdumper case, make compile-first after the dump
 	cp -f temacs$(EXEEXT) bootstrap-emacs$(EXEEXT)
 ifeq ($(DO_CODESIGN),yes)
-	codesign -s - -f bootstrap-emacs$(EXEEXT)
+	/usr/bin/codesign -s - -f bootstrap-emacs$(EXEEXT)
 endif
 endif
